memcached.c

```diff
@@ -7601,76 +7608,104 @@ static void usage(void) {
 #endif
     printf("-o, --extended            comma separated list of extended options\n"
            "                          most options have a 'no_' prefix to disable\n"
-           "   - maxconns_fast:       immediately close new connections after limit\n"
+           "   - maxconns_fast:       immediately close new connections after limit (default: %s)\n"
            "   - hashpower:           an integer multiplier for how large the hash\n"
-           "                          table should be. normally grows at runtime.\n"
+           "                          table should be. normally grows at runtime. (default starts at: %d)\n"
            "                          set based on \"STAT hash_power_level\"\n"
            "   - tail_repair_time:    time in seconds for how long to wait before\n"
            "                          forcefully killing LRU tail item.\n"
            "                          disabled by default; very dangerous option.\n"
            "   - hash_algorithm:      the hash table algorithm\n"
            "                          default is murmur3 hash. options: jenkins, murmur3\n"
-           "   - lru_crawler:         enable LRU Crawler background thread\n"
+           "   - lru_crawler:         enable LRU Crawler background thread \n"
```

Comment:
lru crawler and maintainer threads are enabled by default.

Reply:
Most (all?) boolean options like this have a "no_" counterpart. could just change this to "no_lru_crawler: disable etc"